ipn/ipnlocal/serve.go

```diff
@@ -927,6 +930,38 @@ func encTailscaleHeaderValue(v string) string {
 	return mime.QEncoding.Encode("utf-8", v)
 }
 
+func (b *LocalBackend) addTailscaleGrantHeader(r *httputil.ProxyRequest) {
+	r.Out.Header.Del("Tailscale-App-Capabilities")
```

Comment:
Consider factoring into a constant (can just be a constant inside this function).